error TS2339: Property 'includes' does not exist on type 'string[]'.

## Issue Name
error TS2339: Property 'includes' does not exist on type 'string[]'.

### Summary
build is failing with error 

ERROR in app/datasets/dataset-table/dataset-table.component.ts:200:45 - error TS2339: Property 'includes' does not exist on type 'string[]'.
 200           this.datasets.map(set => set.pid).includes(item.derivedDataset.pid)

### Steps to Reproduce

### Current Behaviour

### Expected Behaviour

### Extra Details

Here you should include details about the system (if it is unique) and possible information about a fix (feel free to link to code where relevant). Screenshots/GIFs are also fine here.

--- a/src/tsconfig.json
+++ b/src/tsconfig.json
@@ -5,7 +5,7 @@
     "declaration": false,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "lib": ["es6", "dom"],
+    "lib": ["es2017", "dom"],
     "mapRoot": "./",
     "module": "es6",
     "moduleResolution": "node",

